ROADMAP.md

```diff
@@ -20,46 +20,56 @@ See also the [ROCm Device Support Wishlist GitHub Discussion](https://github.com
 | ------------ | ----------- | ------------- | ------------- | ------------- |
 | **RDNA4**    | **gfx1201** | ✅            | ✅            | ✅            |
+| **RDNA4**    | **gfx1200** | ✅            | ✅            | ✅            |
 | **RDNA3.5**  | **gfx1151** | ✅            |               |               |
+| **RDNA3.5**  | **gfx1150** | ✅            |               |               |
 | **RDNA3**    | **gfx1102** | ✅            |               |               |
 | **RDNA3**    | **gfx1101** | ✅            |               |               |
 | **RDNA3**    | **gfx1100** | ✅            |               |               |
 | RDNA2        | gfx1036     |               |               |               |
 | RDNA2        | gfx1035     |               |               |               |
 | RDNA2        | gfx1032     |               |               |               |
 | RDNA2        | gfx1030     |               |               |               |
-| GCN5.1       | gfx906      |               |               |               |
+| RDNA1        | gfx1012     | ✅            |               |               |
+| RDNA1        | gfx1011     | ✅            |               |               |
+| RDNA1        | gfx1010     | ✅            |               |               |
+| GCN5.1       | gfx906      | ✅            |               |               |
 
-### HIP Runtime and SDK on Windows
+### ROCm on Windows
 
 Check [windows_support.md](docs/development/windows_support.md) on current status of development.
 
-**AMD Radeon**
+#### AMD Radeon - Windows
 
 | Architecture | LLVM target | Build Passing | Sanity Tested | Release Ready |
 | ------------ | ----------- | ------------- | ------------- | ------------- |
 | **RDNA4**    | **gfx1201** | ✅            |               |               |
+| **RDNA4**    | **gfx1200** | ✅            |               |               |
 | **RDNA3.5**  | **gfx1151** | ✅            | ✅            |               |
+| **RDNA3.5**  | **gfx1150** | ✅            |               |               |
 | **RDNA3**    | **gfx1102** | ✅            |               |               |
 | **RDNA3**    | **gfx1101** | ✅            |               |               |
 | **RDNA3**    | **gfx1100** | ✅            | ✅            |               |
 | RDNA2        | gfx1036     |               |               |               |
 | RDNA2        | gfx1035     |               |               |               |
 | RDNA2        | gfx1032     |               |               |               |
 | RDNA2        | gfx1030     |               |               |               |
-| GCN5.1       | gfx906      |               |               |               |
+| RDNA1        | gfx1012     | ✅            |               |               |
+| RDNA1        | gfx1011     | ✅            |               |               |
+| RDNA1        | gfx1010     | ✅            |               |               |
+| GCN5.1       | gfx906      | ✅            |               |               |
```

Comment:
Added these checks after enabling these nightly builds in https://github.com/ROCm/TheRock/pull/1384